documents-d-archeologie-francaise.csl

```diff
@@ -424,214 +406,164 @@
   </citation>
   <bibliography>
     <sort>
-      <key macro="responsability"/>
+      <key macro="author-short"/>
       <key macro="year-date"/>
       <key macro="title"/>
     </sort>
     <layout>
-      <group prefix="" suffix="" delimiter=" ">
+      <group prefix="" suffix="&#9;" delimiter=" ">
```

Comment:
![csl-daf no tab-block](https://cloud.githubusercontent.com/assets/2558668/12426498/3baca754-beda-11e5-90a6-5aa62bfc243c.PNG)
Here is the result with `
<group display="block" delimiter=" ">
<text macro="author-short"/>
<text macro="year-date" font-weight="bold"/>
</group>
<group display="block" suffix=".&#9;">`